src/amo/components/AddonVersionCard/styles.scss

```diff
@@ -10,6 +10,12 @@
   }
 }
 
+@media only screen and (max-width: 400px) {
```

Comment:
This is a good try, @championshuttler, but there a couple of issues. We try to avoid using media queries and specific widths. The `respond-to` mixin that you can see used above generally takes care of that for us. Also, because you've defined this style for `.Button` in its own section, it will apply to the entire site, not just to this component.

It turns out that we don't actually need to say anything about screen sizes to make this work, because the layout using FlexBox takes care of that for us. All we have to do is define a style inside `InstallButtonWrapper`. I believe that this diff, applied to your patch, will do the trick:

```diff
diff --git a/src/amo/components/AddonVersionCard/styles.scss b/src/amo/components/AddonVersionCard/styles.scss
index f270f7636..cba77f90c 100644
--- a/src/amo/components/AddonVersionCard/styles.scss
+++ b/src/amo/components/AddonVersionCard/styles.scss
@@ -7,12 +7,10 @@

   .InstallButtonWrapper {
     align-self: center;
-  }
-}

-@media only screen and (max-width: 400px) {
-  .Button {
-  width: 100%;
+    .Button {
+      width: 100%;
+    }
   }
 }
```